src/plugins/plot/vue/inspector/PlotOptionsItem.vue

```diff
@@ -0,0 +1,140 @@
+                 :class="getSeriesClass"
+            >
+                <span class="is-status__indicator"
+                      title="This item is missing or suspect"
+                ></span>
+            </div>
+            <div class="c-object-label__name">{{ series.domainObject.name }}</div>
+        </div>
+    </li>
+    <li v-show="expanded"
+        class="c-tree__item menus-to-left"
+    >
+        <ul class="grid-properties">
+            <li class="grid-row">
+                <div class="grid-cell label"
+                     title="The field to be plotted as a value for this series."
+                >Value</div>
+                <div class="grid-cell value">
+                    {{ series.get('yKey') }}
+                </div>
+            </li>
+            <li class="grid-row">
+                <div class="grid-cell label"
+                     title="The rendering method to join lines for this series."
+                >Line Method</div>
+                <div class="grid-cell value">{{ {
+                    'none': 'None',
+                    'linear': 'Linear interpolation',
+                    'stepAfter': 'Step After'
+                }[series.get('interpolate')] }}
+                </div>
+            </li>
+            <li class="grid-row">
+                <div class="grid-cell label"
+                     title="Whether markers are displayed, and their size."
+                >Markers</div>
+                <div class="grid-cell value">
+                    {{ series.markerOptionsDisplayText() }}
+                </div>
+            </li>
+            <li class="grid-row">
+                <div class="grid-cell label"
+                     title="Display markers visually denoting points in alarm."
+                >Alarm Markers</div>
+                <div class="grid-cell value">
+                    {{ series.get('alarmMarkers') ? "Enabled" : "Disabled" }}
+                </div>
+            </li>
+            <li class="grid-row">
+                <div class="grid-cell label"
+                     title="The plot line and marker color for this series."
+                >Color</div>
+                <div class="grid-cell value">
+                    <span class="c-color-swatch"
+                          :style="{
+                              'background': series.get('color').asHexString()
+                          }"
+                    >
+                    </span>
+                </div>
+            </li>
+        </ul>
+    </li>
+</ul>
+</template>
+
+<script>
+export default {
+    inject: ['openmct'],
+    props: {
+        series: {
+            type: Object,
+            default() {
+                return {};
+            }
+        }
+    },
+    data() {
+        return {
+            expanded: false
+        };
+    },
+    computed: {
+        getSeriesClass() {
+            let cssClass = '';
+            let legacyObject = this.openmct.legacyObject(this.series.domainObject);
```

Comment:
No legacy API please. Let's add a method to the Object API to determine if a given object path is to a link. I think the logic basically is "is the `location` property equal to the ID of the parent of this object"